### ISSUE TITLE
Touchscreen doesn't work after long suspend

### ISSUE THREAD

**Issue by thebombsauce** [koreader/koreader#1943]
On the latest nightly of koreader if I go to a book, suspend the book and then wait for awhile when I come back and turn on the book the touchscreen doesn't work and I can't interact with anything.

**Comment by NiLuJe** [comment]
On which device/FW version, and how do you launch KOReader?

**Comment by thebombsauce** [comment]
I'm on a kobo aura h20 running 3.19 and I launch it through kobo start menu

**Comment by houqp** [comment]
What is the last working version for you? When you turn the device back on, do you see any change in the screen/frontlight?

**Comment by thebombsauce** [comment]
I can't remember but this bug has been around for a few versions that I've updated past to try and fix it, and yeah the front light does turn back on and the text does show, so I'm not a coder but I assume that means something that calls the touchscreen to start working again after suspend just doesn't get called.

**Comment by houqp** [comment]
@thebombsauce were you able to reproduce this issue consistently? @NiLuJe maybe we need to enable your `FROM_NICKEL` code path even for cases.

**Comment by holee16** [comment]
Similar error occurred with my KOBO aura H2O (KSM8 and nickel 3.19.5761).

I updated koreader from stable 2015.11 to nightly-20160323 manually.
(deleted koreader folder and copied nightly koreader folder to .add folder) 

After even very short suspend, touch input didn't work. 
Pushed power button, went into screensaver, and pushed power button again, now out of suspend mode, but no touch input. no response. Tried several times. Still no touch response.

Now I reinstalled stable 2015.11 version again. Things look quite stable.
